debug-reindexing.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+
+function wait_weaviate() {
```

Comment:
**suggestion:** 
we can replace that block by `source common.sh` and that will include those 2 func by default and also trap and log the containers in case of any `exit 1`